custom/active-speaker/components/SpeakerView/SpeakerTile/SpeakerTile.js

```diff
@@ -1,29 +1,29 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import Tile from '@custom/shared/components/Tile';
 import { DEFAULT_ASPECT_RATIO } from '@custom/shared/constants';
 import { useResize } from '@custom/shared/hooks/useResize';
+import defaultTheme from '@custom/shared/styles/defaultTheme';
 import PropTypes from 'prop-types';
 
 const MAX_RATIO = DEFAULT_ASPECT_RATIO;
 const MIN_RATIO = 4 / 3;
 
-export const SpeakerTile = ({ participant, screenRef }) => {
+export const SpeakerTile = ({ sessionId, screenRef }) => {
   const [ratio, setRatio] = useState(MAX_RATIO);
   const [nativeAspectRatio, setNativeAspectRatio] = useState(null);
   const [screenHeight, setScreenHeight] = useState(1);
 
   const updateRatio = useCallback(() => {
-    const rect = screenRef.current?.getBoundingClientRect();
-    setRatio(rect.width / rect.height);
-    setScreenHeight(rect.height);
+    if (!screenRef.current) return;
+    const { height, width } = screenRef.current.getBoundingClientRect();
+    setRatio(width / height);
+    setScreenHeight(height);
   }, [screenRef]);
 
   useResize(() => {
     updateRatio();
   }, [updateRatio]);
 
-  useEffect(() => updateRatio(), [updateRatio]);
-
```

Comment:
why remove this? it looks like it needs to be called on render from the looks of it but maybe it's unnecessary?